CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
 ### Added
 
 - Coveralls support (#149)
+- Add public API with a request object for Select/Update/Upstream (#126)
```

Comment:
We usually add changelog entries to the end of the list.

Reply:
Oh, it looks like an order by an issue number. Thank you, I got it.